Sources/AsyncHTTPClient/FoundationExtensions.swift

```diff
@@ -54,3 +54,13 @@ extension HTTPClient.Cookie {
         )
     }
 }
+
+extension HTTPClient.Body {
+    /// Create and stream body using `Data`.
+    ///
+    /// - parameters:
+    ///     - bytes: Body `Data` representation.
+    public static func data(_ data: Data) -> HTTPClient.Body {
+        return bytes(data)
```

Comment:
Yeah, I know this is a ridiculous nit, but:

```suggestion
        return self.bytes(data)
```

Reply:
Done